examples/file-sharing/src/network.rs

```diff
@@ -349,7 +349,8 @@ impl EventLoop {
                     self.swarm
                         .behaviour_mut()
                         .kademlia
-                        .add_address(&peer_id, peer_addr.clone());
+                        .add_address(&peer_id, peer_addr.clone())
+                        .unwrap();
```

Comment:
Could you please use `expect` here, so that when it fails, it is clear what has happened

Reply:
Done